### ISSUE TITLE
Graph

### ISSUE THREAD

**Issue by francnuec** [ra0o0f/arangoclient.net#63]
Please, I don't seem to have it figured out but how do I use Arango's graph querying/traversal capabilities with this client and/or LINQ?

**Comment by ra0o0f** [comment]
@francnuec  please take a look here:

https://github.com/ra0o0f/arangoclient.net/tree/next/src/ArangoDB.Client.Examples/Linq

**Comment by francnuec** [comment]
Thank you. Seeing the examples folder really helped me a lot.

However, say I have this neo4j cypher command:
MATCH (n:Actor)-[r:ACTED_IN]->(n2:Movie { title: 'The Matrix' })
RETURN n, r, n2

How can I achieve same with this ArangoDB client especially with LINQ and its where clauses too to filter. The Traversal command here only seems to consider just a Vertex, and an Edge. I want to know how I can match a particular complete relationship (vertex-edge-vertex) with conditions on one of the vertices.

I'm assuming your knowledge of neo4j. I'm sorry if wrong.

**Comment by Hunter21007** [comment]
Yes i agree, actually both sides of the edge have to be same type.
It would be nice to define more like START-EDGE-END (I do not know how to deal with graphs which involves more different types)

But in the basic usage scenario this would be very helpfull

**Comment by Hunter21007** [comment]
But i see, this is not easy, because there no any type information in the query result. :(
All vertices are in the same collection
